testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -278,6 +279,8 @@
 XYZ = resource_filename(__name__, 'data/2r9r-1b.xyz')
 XYZ_mini = resource_filename(__name__, 'data/mini.xyz')
 XYZ_five = resource_filename(__name__, 'data/five.xyz')
```

Comment:
https://github.com/mmattaNU/mdanalysis/blob/tinker_support/testsuite/setup.py#L207

We've got `coordinates/*` right now, so should be fine